<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="DESKTOP-VF6R5DT" failures="1" tests="3" name="Default test" time="9.266" errors="0" timestamp="2025-04-26T03:39:36 EET">
  <testcase classname="MyNewClass" name="AutoCompleteTest" time="7.302"/>
  <testcase classname="MyNewClass" name="StaticList" time="0.346"/>
  <testcase classname="MyNewClass" name="CheckboxTest" time="0.618">
    <failure type="java.lang.AssertionError" message="The Checkboxbutton3 does not select expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The Checkboxbutton3 does not select expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at MyNewClass.CheckboxTest(MyNewClass.java:160)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- CheckboxTest -->
  <testcase classname="MyNewClass" name="DynamicDropDown" time="0.0">
    <ignored/>
  </testcase> <!-- DynamicDropDown -->
  <testcase classname="MyNewClass" name="PrintDynamicDropDown" time="0.0">
    <ignored/>
  </testcase> <!-- PrintDynamicDropDown -->
  <testcase classname="MyNewClass" name="SelectSecondButtonOfCheckbox" time="0.0">
    <ignored/>
  </testcase> <!-- SelectSecondButtonOfCheckbox -->
  <testcase classname="MyNewClass" name="CheckboxTest1" time="0.0">
    <ignored/>
  </testcase> <!-- CheckboxTest1 -->
  <testcase classname="MyNewClass" name="StaticDropDown" time="0.0">
    <ignored/>
  </testcase> <!-- StaticDropDown -->
  <testcase classname="MyNewClass" name="RadioButton" time="0.0">
    <ignored/>
  </testcase> <!-- RadioButton -->
</testsuite> <!-- Default test -->
